core/common/src/AnalysisStyle.ts

```diff
@@ -8,76 +8,263 @@
 
 import { Range1d, Range1dProps } from "@bentley/geometry-core";
 import { ThematicGradientSettings, ThematicGradientSettingsProps } from "./ThematicDisplay";
+import { Gradient } from "./Gradient";
 
-/** Properties for display of analysis data

```

Comment:
Running into following test build error:
==[ core-full-stack-tests ]======================================[ 65 of 88 ]==
src/frontend/standalone/ViewportChangedEvents.test.ts(236,131): error TS2345: Argument of type '{ inputName: string; }' is not assignable to parameter of type 'AnalysisStyleProps'.
Object literal may only specify known properties, and 'inputName' does not exist in type 'AnalysisStyleProps'.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! core-full-stack-tests@ build:tests: tsc 1>&2 && npm run webpack:frontend
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the core-full-stack-tests@ build:tests script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR! C:\Users\Mark.Schlosser\AppData\Roaming\npm-cache_logs\2021-06-21T18_56_12_547Z-debug.log
Returned error code: 2
"core-full-stack-tests" failed to build.

Reply:
@pmconne Feel free to resolve this conversation on your own to get this merged in once all in good shape.